tee-worker/litentry/core/credentials/src/generic_discord_role/mod.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020-2023 Trust Computing GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{
+	assertion_logic::{AssertionLogic, Op},
+	Credential,
+};
+use litentry_primitives::GenericDiscordRoleType;
+
+// ContestLegend / ContestPopularity / ContestParticipant
+// (type, description)
+const VC_LITENTRY_GENERIC_DISCORD_ROLE_INFOS: [(&str, &str); 3] = [
+	("Litentry & Contest Legend", "You got the Top Award of community contest."),
```

Comment:
It may be named `GenericDiscrodRole` but from description it looks like its bound to some kind of community contest. I think that description should also be generic, if can't then vc/assertion should be renamed.

Reply:
Yeah, I also think so. The name is bit confusing.